src/main/java/net/snowflake/client/core/SFSession.java

```diff
@@ -100,7 +100,7 @@ public class SFSession extends SFBaseSession {
   // client to log session metrics to telemetry in GS
   private Telemetry telemetryClient;
   private SnowflakeConnectString sfConnStr;
-  private QueryContextCache qcc;
+  private QueryContextCache qcc; 
```

Comment:
// The cache of query context sent from Cloud Service.

Reply:
added this comment